### PR TITLE
Move regex

### PR DESCRIPTION
**Pull request by levBagryansky**
@Graur , please review
I copied regex from eo according to #7

### CODE REVIEW

File: src/main/eo/org/eolang/txt/regex.eo [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# The MIT License (MIT)
+#
+#  2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/eo/org/eolang/txt/regex.eo:3]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/main/java/EOorg/EOeolang/EOtxt/EOregex$EOcompile.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/java/EOorg/EOeolang/EOtxt/EOregex$EOcompile.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/main/java/EOorg/EOeolang/EOtxt/EOregex$EOmatch.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/main/java/EOorg/EOeolang/EOtxt/EOregex$EOmatch.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/test/eo/org/eolang/txt/regex-tests.eo [hunk 4]
```diff
@@ -0,0 +1,214 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/test/eo/org/eolang/txt/regex-tests.eo:3]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/test/java/EOorg/EOeolang/EOtxt/EOregexEOcompileTest.java [hunk 5]
```diff
@@ -0,0 +1,98 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/test/java/EOorg/EOeolang/EOtxt/EOregexEOcompileTest.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

File: src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java [hunk 6]
```diff
@@ -0,0 +1,113 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2022 Eugene Darashkevich
```
Comment (Graur) [inline src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java:4]: @levBagryansky I think here should be `# 2016-2022 Objectionary.com`

**Review by Graur — CHANGES_REQUESTED** [review]

File: src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java (hunk 6 above)
Comment (levBagryansky) [reply src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java:4]: @Graur , 
checkstyle requires  Copyright (c) 2022 Eugene Darashkevich in java files

Comment (levBagryansky) [reply src/test/java/EOorg/EOeolang/EOtxt/EOregexEOmatchTest.java:4]: @Graur I also decided do the same in eo files

**Review by Graur — APPROVED** [review]
@yegor256 Please, have a look

**Comment by levBagryansky** [comment]
@yegor256 , look please

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**